weeklymission2/src/Components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import logo from '../images/logo.svg';
+
+export function Header({ profile, folder }) {
+  return (
+    <>
+      <div className="headerArea">
+        <nav>
+          <div className="navBar">
+            <a href="index.html"
+              ><img src={logo} alt="logo" className="logo"
+            /></a>
+            {profile ?
+              (<div className="profile"><img src={profile.image} alt="userProfileImg" className="profileImg"></img>
+              <span className="userEmail">{profile.email}</span></div>) :
+              (<a href="../Pages/signIn.html" className="btn btnS">로그인</a>)
+            }
```

Comment:
해당 조건 연산자는 작동하지 않습니다. 왜냐하면 부모 컴포넌트에서 useState에 디폴트 객체를 넣어줬기때문에 profile객체는 항상 있는게 됩니다. 차라리 이 조건이 동작하게 하려면 디폴트객체를 줬으니 객체의 프로퍼티가 잘 들어 가 있는지 확인하면 됩니다.

```suggestion
            {profile.email ?
              (<div className="profile"><img src={profile.image} alt="userProfileImg" className="profileImg"></img>
              <span className="userEmail">{profile.email}</span></div>) :
              (<a href="../Pages/signIn.html" className="btn btnS">로그인</a>)
            }
```